EIPS/eip-5625.md

```diff
@@ -0,0 +1,204 @@
+                },
+                "default": {
+                    "type": "string",
+                    "description": "The locale of the default data within the base JSON"
+                },
+                "locales": {
+                    "type": "array",
+                    "description": "The list of locales for which data is available. These locales should conform to those defined in the Unicode Common Locale Data Repository (http://cldr.unicode.org/)."
+                }
+            }
+        },
+        "dStorage": {
+            "type": "object",
+            "required": ["platform", "description", "persistence_mechanism", "challenge_mechanism", "consensus", "dstorage_note"],
+            "properties": {
+                "platform": {
+                    "type": "string",
+                    "description": "dStorage platform name like Swarm, Arweave, Filecoin, Crust, etc"
+                },
+                "description": {
+                    "type": "string",
+                    "description": "A brief description of the dStorage platform"
+                },
+                "persistence_mechanism": {
+                    "type": "string",
+                    "description": "Persistence mechanism or incentive structure of the dStorage platform, like 'blockchain-based', 'contract-based', etc"
+                },
+                "challenge_mechanism": {
+                    "type": "string",
+                    "description": "Challenge mechanism of the dStorage platform, like Arweave's proof-of-access, etc"
+                },
+                "consensus": {
+                    "type": "string",
+                    "description": "Consensus mechanism of the dStorage platform, like PoW, PoS, etc"
+                },
+                "dstorage_note": {
+                    "type": "string",
+                    "description": "A note to prove the storage of the NFT asset on the dStorage platform, like a Filecoin deal id, a Crust place_storage_order transaction hash, etc"
+                }
+            }
+        }
+    }
+}
+```
+
+An example of an [EIP-721](./eip-721.md) NFT asset Metadata JSON file follows. The `dStorage` property contains information about how the NFT asset is stored on Crust Network, which can be verified on https://ipfs-scan.io?cid=QmVvigK7jfLyHcDjnNYr2mAReFW9FCFWf41UNB9tbK8rWw.
+
+
+```json
+{
+	"name": "LAND (13, -86)",
+	"description": "A LAND is a digital piece of real estate in The Sandbox metaverse that players can buy to build experiences on top of",
+	"image": "ipfs://QmVvigK7jfLyHcDjnNYr2mAReFW9FCFWf41UNB9tbK8rWw",
+	"dStorage": {
+		"platform": "Crust",
+    "description": "Crust Network, the Incentive Layer of IPFS",
+    "persistence_mechanism": "contract-based",
+    "challenge_mechanism": "mpow",
+    "consensus": "gpos",
+    "dstorage_note": "0x466ab180124de1718d30cd2e018e7fe013a07c4860674110ccd13e97eb31ae16"
+	}
+}
+```
+
```

Comment:
```suggestion
```
This should either be in the reference implementation section or omitted. If you would prefer your EIP to be merged more quickly, I would recommend just omitting it.